docs/0-what-is/0-introduction.mdx

```diff
@@ -12,264 +12,40 @@ slug: '/'
   src="/img/conduit/on-white-conduit-logo.png"
 />
 
-Conduit is a data integration tool for software engineers, powered by [Meroxa](https://meroxa.io). Its purpose is to
-help you move data from A to B. You can use Conduit to send data from Kafka to
-Postgres, between files and APIs,
-between [supported connectors](/docs/using/connectors/list),
+Conduit is a data streaming tool for software and data engineers, powered
+by [Meroxa](https://meroxa.io). Its purpose is to help you move data from A to
+B. You can use Conduit to send data from Kafka to Postgres, between files and
+APIs, between [supported connectors](/docs/using/connectors/list),
 and [any datastore you can build a plugin for](/docs/developing/connectors/).
 
-It's written in [Go](https://go.dev/), compiles to a binary, and is designed to
-be easy to use and [deploy](/docs/installing-and-running?option=binary).
+It's written in [Go](https://go.dev/), compiles to a single binary, and is
+designed to be easy to use and [deploy](/docs/installing-and-running).
```

Comment:
now that I see this link, I don think we should link `deploy` to this page. It's about installing, so I don't think it makes sense at all.

Reply:
Agreed, I'll change it to `install`.